src/Test.EndToEnd.Baselining/Options/DebugOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using CommandLine;
+
+namespace Test.EndToEnd.Baselining.Options
+{
+    [Verb("debug", HelpText = "Debug Baselining for a specific Series, Log, and Result")]
+    public class DebugOptions : OptionsBase
+    {
+        [Value(1, Required = false, HelpText = "Debug: Series Path (everything under 'Input/'")]
```

Comment:
>everything [](start = 68, length = 10)

Did you mean to write "default: everything under 'Input/'"?

Reply:
No; fixed to make required, fixed to explain the arg is the portion of the series path after Input\